.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  release-e2e-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.18'
+
+    - name: install_kind_binary
+      env:
+      run: |
+        make kind
+
+    - name: start_kind_cluster
+      run: |
+        hack/tools/bin/kind create cluster
+        hack/tools/bin/kind export kubeconfig
+
+    - name: install_kubectl
+      run: |
+        curl -LO "https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl"
+        sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+    - name: Set the release related variables
+      run: |
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+          IMAGE_TAG=${GITHUB_REF#refs/tags/} >> $GITHUB_ENV
+          fi
+
+    - name: install_cert_manager
+      run: |
+        make cert-mgr
+
+    - name: release_install
+      run: |
+        kubectl apply -f https://github.com/operator-framework/rukpak/releases/download/${IMAGE_TAG}/rukpak.yaml
```

Comment:
I think this might be off as we echo the IMAGE_TAG variable we built up in a previous job to github's environment, which can be indexed from the [`env` context](https://docs.github.com/en/actions/learn-github-actions/contexts#env-context):

```
kubectl apply -f https://github.com/operator-framework/rukpak/releases/download/${{ env.IMAGE_TAG }}/rukpak.yaml